pkg/keys/keys.go

```diff
@@ -31,6 +31,9 @@ import (
 // KeyManager implementations must be able to return a crypto.Signer.
 type KeyManager interface {
 	NewSigner(ctx context.Context, keyID string) (crypto.Signer, error)
+
+	Encrypt(ctx context.Context, keyID string, plaintext []byte, aad string) ([]byte, error)
```

Comment:
Can we doc these? Is it valid for aad to be ""?

Reply:
Also, should aad be []byte?

Reply:
went back and forth on it - agree []byte makes more sense.

practically, we'll probably take this in as a base64 env var